core/src/main/java/org/geysermc/geyser/session/GeyserSession.java

```diff
@@ -1642,6 +1643,9 @@ public void sendUpstreamPacketImmediately(BedrockPacket packet) {
      * @param packet the java edition packet from MCProtocolLib
      */
     public void sendDownstreamGamePacket(Packet packet) {
+        if (packet instanceof ServerboundSwingPacket) {
+            geyser.getLogger().error("sending: " + packet.toString());
```

Comment:
Is this meant to be here still?

Reply:
will remove before merge, this could use some testing on other platforms